src/swarm/neo/client/requests/Stats.d

```diff
@@ -57,6 +57,23 @@ public interface IRequestStats
     /// identifying the type of a request.)
     private alias IRequest.FinishedNotifier RequestTypeIdentifier;
 
+    /***************************************************************************
+
+        Checks whether stats for the specified request have ever been collected.
+        This is useful in situations where the user wants to differentiate
+        between a request that has not occurred since the last call to clear()
+        vs a request that has _never_ occurred.
+
+        Params:
+            request_type = identifier for the request to check
+
+        Returns:
+            true if this request has ever occurred
+
+    ***************************************************************************/
+
+    public bool requestHasOccurred ( RequestTypeIdentifier request_type );
```

Comment:
This interface is only implemented in swarm, right? This would mean that this is a new feature, and not a breaking change.

Reply:
Yep. Of course it's the odd situation that anyone who has swarm as a submodule _could_ theoretically implement this interface, but that would be highly unusual.